kitten_options_definition.py

```diff
@@ -103,6 +103,14 @@
 map('SelectColumnar word left',      'select_columnar Alt+Ctrl+Left  select columnar word left')
 map('SelectColumnar word right',     'select_columnar Alt+Ctrl+Right select columnar word right', long_text=long_text)
 
+long_text = '''
+Keys to enable vim-like modal selecting.
+Available modes are "normal", "visual", and "block".
```

Comment:
Available modes do not need to be enumerated in the description because they are listed in the following `map` lines. If you do want to keep this line, please use typographic quotes: `“”` or preferably `‘’`.

Reply:
I also believe you need to regenerate the config parser:
```bash
$ kitty +runpy 'from kitty.conf.generate import main; main()' \
    "$(pwd)/kitten_options_definitions.py"
```

Reply:
Actually, I didn't want to enable these keys by default, since they may be confusing for non-vim users.